modules/nextclade/run/main.nf

```diff
@@ -29,17 +34,14 @@ process NEXTCLADE_RUN {
         run \\
         $args \\
         --jobs $task.cpus \\
-        --input-fasta $fasta \\
```

Comment:
I see that the `fasta` remains as a declare input I guess now is automatically detected by the tool, right?